src/server/campaignd/server.cpp

```diff
@@ -898,30 +1064,113 @@ void server::handle_upload(const server::request& req)
 		}
 
 		const std::string& filename = (*campaign)["filename"].str();
-		data["title"] = (*campaign)["title"];
 		data["name"] = "";
-		data["campaign_name"] = (*campaign)["name"];
-		data["author"] = (*campaign)["author"];
-		data["description"] = (*campaign)["description"];
-		data["version"] = (*campaign)["version"];
-		data["timestamp"] = (*campaign)["timestamp"];
-		data["original_timestamp"] = (*campaign)["original_timestamp"];
-		data["icon"] = (*campaign)["icon"];
-		data["type"] = (*campaign)["type"];
-		data["tags"] = (*campaign)["tags"];
```

Comment:
why did these get removed?

Reply:
To not duplicate the addon meta in the full packs, and this data is never used by the client.

Reply:
hmm, maybe it better to duplicate  this data for robustness though (so that if the config file gets lost one still knows what these files are by looking inside them) not sure. @shikadiqueen an opinion on this?

Reply:
This is a really annoying one because we need to explicitly duplicate almost all of the metadata. It's true there's *some* value in keeping the metadata around (I used it in combination with server logs and a previous backup to restore the add-ons server to a working state a couple of years ago after a major accident), but for the purpose of this PR I find it's an inconvenient distraction. Maybe we can evaluate re-adding it at a later time with cleaner code.